CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
-- [IMPROVED] Add HTTP status code to `Response` objects.
+- [IMPROVED] Added parameter pagination option for views. See `ViewRequest.getResponse(String)`.
+- [IMPROVED] Added HTTP status code to `Response` objects.
```

Comment:
This last line should be a separate commit, doesn't belong with the view pagination stuff.